### ISSUE TITLE
post a file (via API) to a entry that allows multiple files, remove existed relation.

### ISSUE THREAD

**Issue by takahiro-yonei** [strapi/strapi#3048]
**Informations**
- **Node.js version**: 10.15.1
- **NPM version**: 6.4.1
- **Strapi version**: 3.0.0-alpha.24
- **Database**: Postgres
- **Operating system**: macOS Sierra (10.12.6)

**What is the current behavior?**

I defined a model with multiple files, and upload a file to the entry via API ( `/upload` ) twice.
I expected the entry has two related files, but the entry has only one.

**Steps to reproduce the problem**

1. define a model with a field `files` with multiple uploads allowed.
2. create a entry.
3. post a file to the entry.
    * `curl -X POST -F 'files=@xxxx.png' -F 'refId=(entry id)' -F 'ref=(model)' -F 'field=files' http://localhost:1337/upload'`
4. post another file to the same entry.
    * `curl -X POST -F 'files=@zzzz.png' -F 'refId=(entry id)' -F 'ref=(model)' -F 'field=files' http://localhost:1337/upload'`
5. The entry has only one related file (zzzz.png)
    * There is only one record in `upload_file_morph` table.

**What is the expected behavior?**

After step 4, the entry has two related files. (xxxx.png and zzzz.png)
In other words, there are two records in `upload_file_morph` table.

If I define a model with **single** upload, it's OK.
But If I define a model with **multiple** uploads allowed, I expected not to remove existed relation.

**Suggested solutions**

Please add a file to the entry, and not remove existed relation.

**Comment by lauriejim** [comment]
Hello @takahiro-yonei !

I reproduce your steps and it works on my side.
Both files are uploaded and I can see both file attached to my entry.

I was tested it with version `3.0.0-alpha.25.2`

Can you try with a fresh install of this version and tell me if the issue persist.

**Comment by takahiro-yonei** [comment]
Hi, @lauriejim 

Thank you for your reproduce.
I will confirm with the latest version.

Thank you so much again.

**Comment by takahiro-yonei** [comment]
@lauriejim 

I tried my steps with `3.0.0-alpha.25.2`.
It works completely fine!

Thank you so much.
And I really appreciate for your great works!

**Comment by JRakhimov** [comment]
Strapi version: 3.0.0-alpha.25.2
Database: Postgres
Operating system: Windows 10

I am doing the same thing but with strapi-javascript-sdk and dont get 2 files, after 2 requests I get only the last one. Any ideas?

```js
const testData = new FormData();

testData.append("files", file);
// if I add one more testData.append("files", file); - I get 2 files, but i need 2 different requests
testData.append("ref", "**");
testData.append("refId", ***);
testData.append("field", "****");

console.log(await strapi.upload(testData));
```

@lauriejim @takahiro-yonei

**Comment by takahiro-yonei** [comment]
@JRakhimov 
Sorry for too late response.

Please check strapi logs.
Is there any `insert` log twice? or is there any `delete` log?

---
by the way, I try again with strapi 3.0.0-beta14.
it works completely.
